xml/art_raspberry-pi.xml

```diff
@@ -665,14 +569,14 @@ Password:
     </step>
     <step>
      <para>
-      Select the time zone by using the arrow keys or first letter.
+      Select the time zone using the arrow keys.
```

Comment:
Are short cuts no longer working, or do you consider it too advanced to mention in this guide?

Reply:
I'd like to keep it simple for the user. Let's give them one way of doing things, instead of bamboozling them with options. Also, explaining all the possible ways of navigation is not what the quick start guide is for. :-)